packages/client/src/views/Performance/FieldAgentList.tsx

```diff
@@ -160,11 +161,10 @@ function FieldAgentListComponent(props: IProps) {
     location: { search },
     offlineOffices
   } = props
-  const { locationId, timeStart, timeEnd } = parse(
+  const { event, locationId, timeStart, timeEnd } = parse(
```

Comment:
```suggestion
  const { event = Event.Birth, locationId, timeStart, timeEnd } = parse(
```
This way we can remove the optional chaining for event right?